components/prism-webidl.js

```diff
@@ -0,0 +1,12 @@
+Prism.languages.webidl = Prism.languages.extend('c', {
+	'keyword': /\b(any|attribute|boolean|byte|ByteString|callback|const|deleter|dictionary|DOMString|double|enum|float|FrozenArray|getter|identifier|implements|Infinity|inherit|integer|interface|iterable|legacycaller|long|maplike|namespace|NaN|null|object|octet|optional|or|other|partial|readonly|required|sequence|serializer|setlike|setter|short|static|string|stringifier|typedef|unrestricted|unsigned|USVString|void)\b/,
+	'boolean': /\b(true|false)\b/,
+	'operator': /-|:|=|\?|\*|\//
+});
+
+Prism.languages.insertBefore('webidl', 'keyword', {
+	'class-name': {
+		pattern: /(class\s+)[a-z0-9_]+/i,
+		lookbehind: true
+	}
+});
```

Comment:
Just add the "class-name" token before the keyword token. Due to the hyphen, you'd need to surround it in quotes, but apart from that, it's just a normal property.

As for the dependencies, I don't mind which one you inherit from, just be consistent between your code and the components declaration, otherwise people will get a download that doesn't work. Perhaps look over their examples to decide which of the three is closer to WebIDL?